xarray/tests/test_distributed.py

```diff
@@ -142,6 +142,20 @@ def test_dask_distributed_rasterio_integration_test(loop):
                 assert_allclose(actual, expected)
 
 
+@requires_cfgrib
+def test_dask_distributed_cfgrib_integration_test(loop):
+    with cluster() as (s, [a, b]):
+        with Client(s['address'], loop=loop) as c:
+            with open_example_dataset('example.grib',
+                                      engine='cfgrib',
+                                      chunks={'time': 1}) as ds:
```

Comment:
This looks reasonable to me. I assume `example.grib` has multiple time points in it?

Reply:
Yes, three.